core/modules/replica/MessengerConnector.h

```diff
@@ -0,0 +1,523 @@
+/*
+ * LSST Data Management System
+ * Copyright 2017 LSST Corporation.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+#ifndef LSST_QSERV_REPLICA_MESSENGERCONNECTOR_H
+#define LSST_QSERV_REPLICA_MESSENGERCONNECTOR_H
+
+/// MessengerConnector.h declares:
+///
+/// class MessengerConnector
+/// (see individual class documentation for more information)
+
+// System headers
+#include <functional>
+#include <list>
```

Comment:
I think `list` is not used in this file?

Reply:
Actually, it's used at line:

507:    /// The queue of requests
508:    std::list<WrapperBase::Ptr> _requests;